From: Wolfram Sang <w.sang@pengutronix.de>
Subject: fix warnings

This Patch fixes the following warnings:

 chkfont.c:68: warning: incompatible implicit declaration of built-in function 'malloc'
 chkfont.c: In function 'usageerr':
 chkfont.c:103: warning: incompatible implicit declaration of built-in function 'exit'
 chkfont.c: In function 'checkit':
 chkfont.c:226: warning: incompatible implicit declaration of built-in function 'exit'
 chkfont.c:260: warning: incompatible implicit declaration of built-in function 'free'
 chkfont.c: In function 'main':
 chkfont.c:458: warning: incompatible implicit declaration of built-in function 'free'
 chkfont.c:460: warning: incompatible implicit declaration of built-in function 'exit'

Signed-off-by: Wolfram Sang <w.sang@pengutronix.de>
---
# 20091221 rsc: mailed to info@figlet.org

 chkfont.c |    1 +
 1 file changed, 1 insertion(+)

Index: chkfont.c
===================================================================
--- chkfont.c.orig
+++ chkfont.c
@@ -1,4 +1,5 @@
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 #include <ctype.h>
 
